atomicapp/nulecule/main.py

```diff
@@ -147,10 +147,19 @@ def genanswers(self, dryrun=False, answers_format=None, **kwargs):
         self.answers_format = answers_format or ANSWERS_FILE_SAMPLE_FORMAT
 
         # Check to make sure an answers.conf file doesn't exist already
-        answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        if location is None:
+            answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        else:
+            answers_file = os.path.join(location, ANSWERS_FILE)
```

Comment:
I'm a bit confused in regards to self.app_path.

So if inContainer() comes back as true, we'd use the /host/ path, correct?